Source/PLCrashMachExceptionPortSet.m

```diff
@@ -106,20 +106,14 @@ - (id) initWithAsyncSafeRepresentation: (plcrash_mach_exception_port_set_t) asyn
         }
     }
 
-    _state_set = [stateResult retain];
+    _state_set = stateResult;
     _asyncSafeRepresentation = asyncSafeRepresentation;
 
     return self;
 }
 
-- (void) dealloc {
-    [_state_set release];
-    [super dealloc];
-}
-
-
 // from NSFastEnumeration protocol
-- (NSUInteger) countByEnumeratingWithState: (NSFastEnumerationState *) state objects: (id *) stackbuf count: (NSUInteger) len {
+- (NSUInteger) countByEnumeratingWithState: (NSFastEnumerationState *) state objects: (id __unsafe_unretained _Nullable [_Nonnull]) stackbuf count: (NSUInteger) len {
```

Comment:
What does `_Nullable [_Nonnull]` mean?

Reply:
We don't know either. This signature was updated for compliance with `NSFastEnumeration` protocol: 
![image](https://user-images.githubusercontent.com/34677812/80496468-eab10180-8971-11ea-963a-e8fbc61ed636.png)

Reply:
I guess this means we can leave only one option here...